lib/elixir/lib/enum.ex

```diff
@@ -932,7 +932,8 @@ defmodule Enum do
 
   @doc """
   Returns a new enumerable appending the result of invoking `fun` on
-  each corresponding item of `enumerable`.
+  each corresponding item of `enumerable` and flattening
+  the given results (only one level deep)..
```

Comment:
Maybe it's funny and small change, but function name is flat_map and comment doesn't say a thing about flattening. It's inaccurate.